projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -643,13 +655,16 @@ class="jp-carousel-comment-form-button"
 	}
 
 	/**
-	 * Unless AMP is enabled and active, sets the 'in_gallery' option to true and returns $output. Otherwise, returns $output.
+	 * Sets the "in_gallery" flag when the first gallery is encountered (unless in AMP mode).
+	 *
+	 * @deprecated since $$next-version$$
 	 *
-	 * @param string $output Output which includes whether or not to set in the gallery.
+	 * @param string $output Gallery shortcode output. Passed through unchanged.
 	 *
 	 * @return string
 	 */
 	public function set_in_gallery( $output ) {
+		_deprecated_function( __FUNCTION__, '10.7' );
```

Comment:
Now that I look at this again, we probably don't want to `_deprecated_function()` this until we remove the call on line 108.
```suggestion
```

Reply:
Ok. Should we also remove the `@deprecated` line in the function comment - or remove the call on 108 instead? Or is removing the call best left for another time?

Reply:
Sounds good. I'll do a separate commit for that additional `@deprecated` line and commit this existing suggestion.